gooey/_tmp/example_progress_bar_4.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+from time import sleep
+from gooey import Gooey, GooeyParser
+
+
+@Gooey(progress_regex=r"^progress: (-?\d+)%$",
+       progress_animation=False)
```

Comment:
@chriskiehl Please see [this](http://stackoverflow.com/questions/5332616/disabling-net-progressbar-animation-when-changing-value). Windows 7 progress bar animation sometimes too slow and lagging from real progress if it changes fast. I've got a log for 20% of the progress.